querybook/webapp/components/DataDocViewersList/DataDocViewersList.scss

```diff
@@ -43,6 +36,12 @@
         overflow-y: auto;
         padding-bottom: 0rem;
         padding-top: 0rem;
+        border-top: 1px solid var(--border-color);
+
+        > div {
```

Comment:
instead of per div, why not just apply the padding in wrapper?

Reply:
bc then the border wouldn't go all the way to the ends